core-api/src/main/java/com/optimizely/ab/odp/ODPSegmentManager.java

```diff
@@ -67,7 +67,7 @@ public List<String> getQualifiedSegments(ODPUserKey userKey, String userValue) {
 
     public List<String> getQualifiedSegments(ODPUserKey userKey, String userValue, List<ODPSegmentOption> options) {
         if (!odpConfig.isReady()) {
-            logger.warn("ODP Config not ready. apiHost and/or apiKey null. Returning Empty list");
+            logger.error("ODP is not enabled.");
```

Comment:
I mean all segments error should come with "Audience segments fetch failed (<more hint here>)". So in this case, "Audience segments fetch failed (ODP is not enabled)". These common messages will be used for all SDKs.

Reply:
Done!